app/src/main/java/org/wikipedia/main/MainFragment.kt

```diff
@@ -253,13 +253,13 @@ class MainFragment : Fragment(), BackPressedHandler, FeedFragment.Callback, Hist
         }
     }
 
-    override fun onFeedSelectPageWithAnimation(entry: HistoryEntry, sharedElements: Array<Pair<View, String>>) {
```

Comment:
It would be better if it is renamed on the `interface` side, which is following:
```
fun onFeedSelectPageWithAnimation(entry: HistoryEntry, sharedElements: Array<Pair<View, String>>)
```